ports/windows/Makefile

```diff
@@ -84,6 +84,11 @@ ifneq ($(FROZEN_MANIFEST),)
 CFLAGS += -DMICROPY_QSTR_EXTRA_POOL=mp_qstr_frozen_const_pool -DMICROPY_MODULE_FROZEN_MPY=1 -DMPZ_DIG_SIZE=16
 endif
 
+ifeq ($(MSYSTEM),MINGW32)
```

Comment:
We should be able to come up with a better check here. Really we should be checking for GCC and x86 target.

Reply:
Actually, I think these options are enabled by default for most builds of GCC already, so maybe this check is correct. For example, I just checked ` x86_64-w64-mingw32-gcc -Q --help=target | grep sse` in cygwin and it is already enabled.